pkgs/top-level/all-packages.nix

```diff
@@ -17026,8 +17029,8 @@ with pkgs;
     nativeTools = stdenv.targetPlatform == stdenv.hostPlatform && stdenv.cc.nativeTools or false;
     nativeLibc = stdenv.targetPlatform == stdenv.hostPlatform && stdenv.cc.nativeLibc or false;
     nativePrefix = stdenv.cc.nativePrefix or "";
-
-    noLibc = (self.libc == null);
+    propagateDoc = if self.nativeTools then false else ((stdenv.cc ? man) && (bintools ? man));
+    noLibc = !self.nativeLibc && (self.libc == null);
```

Comment:
@Ericson2314 what does nativeLibc mean?  Could we have a comment or something somewhere explaining it?

Reply:
Using the psychic onset from standing on a mountaintop in the crisp spring air (in front of my computer, at home) I determined that you really don't want this diff. It's gone now.